src/Core/Domain/CustomerService/Command/BulkDeleteCustomerThreadCommand.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Domain\CustomerService\Command;
+
+use PrestaShop\PrestaShop\Core\Domain\CustomerService\Exception\CustomerServiceException;
+use PrestaShop\PrestaShop\Core\Domain\CustomerService\ValueObject\CustomerThreadId;
+
+class BulkDeleteCustomerThreadCommand
+{
+    /**
+     * @var CustomerThreadId[]
+     */
+    private $customerThreadIds;
+
+    /**
+     * @param array<int, int> $customerThreadIds
+     *
+     * @throws CustomerServiceException
+     */
+    public function __construct(array $customerThreadIds)
+    {
+        $this->setCustomerThreadIds($customerThreadIds);
+    }
+
+    /**
+     * @return CustomerThreadId[]
+     */
+    public function getCustomerThreadIds(): array
+    {
+        return $this->customerThreadIds;
+    }
+
+    /**
+     * @param array<int, int> $customerThreadIds
+     *
+     * @throws CustomerServiceException
+     */
+    public function setCustomerThreadIds(array $customerThreadIds)
+    {
+        foreach ($customerThreadIds as $customerThreadId) {
+            $this->customerThreadIds[] = new CustomerThreadId($customerThreadId);
+        }
+    }
```

Comment:
```suggestion
        return $this;
    }
```

Reply:
okay here i think mistake was to leave it public, it is only used in __construct so i changed it to private and added void return

Reply:
Agreed, this way now is fine